content/hi/docs/setup/production-environment/turnkey-solutions.md

```diff
@@ -0,0 +1,12 @@
+---
+title:  टर्नकी क्लाउड सॉल्यूशंस
+content_type:  अवधारणा
```

Comment:
```suggestion
content_type: concept
```

Reply:
@ShivamTyagi12345 Please commit the change as content_type should be concept and not translated. Conversation will be automatically resolved then